scripts/go.bash

```diff
@@ -21,7 +21,7 @@ if [[ -z $LOGZIO_TOKEN ]]; then
     exit 1
 fi
 
-if ! [[ -z $AWS_ACCESS_KEY && -z $AWS_SECRET_KEY ]]; then
+if ! [[ -z $RDS_LOG_FILE && -z $RDS_SLOW_LOG_FILE && -z $RDS_ERROR_LOG_FILE ]]; then
```

Comment:
no need for that

Reply:
If we want to use IAM roles, no values are passed for _AWS_ACCESS_KEY_ & _AWS_SECRET_KEY_. Thus, this condition would be true and configure_rds would never be called. Therefore, another characteristic is required to indicate "I want to ship RDS logs".

Reply:
why do you need any condition, i think you can just remove this condition, and let the configure_rds to be called. 
RDS_LOG_FILE, RDS_SLOW_LOG_FILE, RDS_ERROR_LOG_FILE will be validated on the sync_rds call, am i missing some thing ?